app/src/main/java/com/example/studentarena/fragments/ComposeFragment.java

```diff
@@ -81,11 +81,11 @@ public void onClick(View v) {
         btnSubmit.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                String description = etDescription.getText().toString();
-                String contact = etContactinfo.getText().toString();
-                String title = etTitle.getText().toString();
-                String price = etPrice.getText().toString();
-                if (description.isEmpty()){
+//                String description = etDescription.getText().toString();
+//                String contact = etContactinfo.getText().toString();
+//                String title = etTitle.getText().toString();
+//                String price = etPrice.getText().toString();
```

Comment:
i'm still not really understanding why you prefer to keep so many commented-out lines like these. if you want to keep them in case you need to do something similar later, it'd probably be better to take notes in a document somewhere; keeps the code cleaner, and you can cmd+f to search the document whereas with these code comments you have to remember "ahh, i think i used to do that in ComposeFragment.java"

i leave commented-out code sometimes if i expect to uncomment that code in a future commit, like if it uses a feature i haven't implemented yet. otherwise i delete stuff like this, and i think it'd be easier for me to review if you did as well because you have a lot of commented-out code